cobalt/media/decoder_buffer_allocator.cc

```diff
@@ -32,25 +32,17 @@ namespace {
 
 const bool kEnableAllocationLog = false;
 
-const size_t kAllocationRecordGranularity = 512 * 1024;
 // Used to determine if the memory allocated is large. The underlying logic can
 // be different.
 const size_t kSmallAllocationThreshold = 512;
 
 }  // namespace
 
 DecoderBufferAllocator::DecoderBufferAllocator()
-    : using_memory_pool_(SbMediaIsBufferUsingMemoryPool()),
```

Comment:
Forcing memory pool may affect performance on platforms where the system allocator is more optimized.

Reply:
Ack